egui/src/widgets/button.rs

```diff
@@ -55,6 +56,28 @@ impl Button {
             frame: None,
             wrap: None,
             min_size: Vec2::ZERO,
+            image: None,
+        }
+    }
+
+    #[allow(clippy::needless_pass_by_value)]
+    pub fn image_and_text(
+        texture_id: TextureId,
+        size: impl Into<Vec2>,
+        text: impl ToString,
```

Comment:
```suggestion
        text: impl Into<WidgetText>,
```

(please rebase on latest `master`)

Reply:
Re-based to use the latest master version.